packages/typespec-azure-resource-manager/lib/models.tsp

```diff
@@ -376,4 +376,20 @@ model EncryptionProperty {
   encryption: EncryptionConfiguration;
 }
 
+/**
+ * Model representing the standard `zones` envelope property for a resource.
+ * Spread this model into a resource Model if the resource supports ARM `zones`.
+ * @example
+ * ```typescript
+ * model Foo is TrackedResource<FooProperties> {
+ *   ...AvailabilityZonesProperty;
+ * }
+ * ```
+ */
+@doc("Availability zones envelope.")
```

Comment:
sorry should have commented here too, I don't think this `@doc` is necessary as this model is really only meant to be used as spread.